addon/components/power-select-multiple.ts

```diff
@@ -25,7 +24,7 @@ export default class PowerSelectMultiple extends Component<PowerSelectMultipleAr
   }
 
   get triggerComponent() {
-    return this.args.triggerComponent || PowerSelectTriggerComponent;
+    return this.args.triggerComponent || 'power-select-multiple/trigger';
```

Comment:
No, this is definitely a regression. Embroider prefers the class here.

Reply:
Yes, I had it with the class but this [test](https://github.com/cibernox/ember-power-select/blob/c2f917e1b03aef0b85a0a09b12f9e05e9b04749d/tests/integration/components/power-select/customization-with-extending-components-test.js#L253) failed not sure how to handle this case